### DISCUSSION TITLE
RGB instead of Grayscale

### DISCUSSION THREAD

**Discussion by PietroVitiello** [zju3dv/LoFTR#262] [Q&A] [answered]
Hi, I was just wondering whether there was any reason why the backbone works with grayscale images rather than RGB. Have you noticed that the additional information worsened the training?

Thanks for the great work!

**Comment by zehongs** [comment, marked as answer]
The main reason is that the experimental results are similar. And most local descriptors in literature use grayscale, so we use grayscale images as inputs.

**Reply by PietroVitiello** [reply]
I understand, thank you very much for the answer!
